### DISCUSSION TITLE
What Linux for OpenHantek6022?

### DISCUSSION THREAD

**Discussion by MaMiKoe** [OpenHantek/OpenHantek6022#263] [Q&A] [answered]
I am absolutely not familiar with Linux. So I would like to know:
Does OpenHantek6022 run on 32bit-Linux-Systems?
What Linux should I (as said absolute newbie and Linux-DAU) install for use with OpenHantek6022?

**Comment by bill-in-iowa** [comment]
Since no one else is answering you, I thought I would give it a shot. The linux distro doesnt really matter, they all use the linux kernel. Long as the kernel is up to date , it should run ok.

**Comment by Ho-Ro** [comment, marked as answer]
> Does OpenHantek6022 run on 32bit-Linux-Systems?

The provided packages are 64bit only, on a 32bit system you must build your own package.
If you do not have any experience I would vote strictly against the 32bit-way.

As written above, every debian should work. Ubuntu would be a good starting point - it has a huge user base and a lot of info regarding ubuntu is available in the net.
